projects/plugins/jetpack/tests/php/modules/masterbar/test-class-css-nudge-customize-control.php

```diff
@@ -11,15 +11,42 @@
 require_once ABSPATH . WPINC . '/class-wp-customize-control.php';
 require_once ABSPATH . WPINC . '/class-wp-customize-manager.php';
 
-require_once JETPACK__PLUGIN_DIR . 'modules/masterbar/nudges/bootstrap.php';
-
 /**
  * Class Test_CSS_Nudge_Customize_Control
  */
 class Test_CSS_Nudge_Customize_Control extends \WP_UnitTestCase {
 
+	/**
+	 * File path for loading the required deprecated file.
+	 *
+	 * @var string
+	 */
+	private static $deprecated_file_path = JETPACK__PLUGIN_DIR . 'modules/masterbar/nudges/bootstrap.php';
+
+	/**
+	 * Runs before each test.
+	 *
+	 * @return void
+	 */
+	public function set_up() {
+		parent::set_up();
+
+		if ( ! in_array( self::$deprecated_file_path, get_included_files(), true ) ) {
+			$this->setExpectedDeprecated( self::$deprecated_file_path );
+			$this->setExpectedDeprecated( 'Automattic\Jetpack\Dashboard_Customizations\load_bootstrap_on_init' );
+			// phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.NotAbsolutePath -- It's absolute in the class property definition.
+			require_once self::$deprecated_file_path;
+		}
+
+		do_action( 'init' );
+	}
+
 	/**
 	 * Check if the HTML for the nudge section is properly generated.
+	 *
+	 * @expectedDeprecated Automattic\Jetpack\Dashboard_Customizations\register_css_nudge_control
+	 * @expectedDeprecated Automattic\Jetpack\Dashboard_Customizations\CSS_Nudge_Customize_Control::render_content
+	 * @expectedDeprecated Automattic\Jetpack\Dashboard_Customizations\CSS_Nudge_Customize_Control::render_content
```

Comment:
Does this line need to be duplicated?

Reply:
Nice catch! Fixed via https://github.com/Automattic/jetpack/pull/38109/commits/f52923c4ed177956286a0f44f50ba75e8fbc6ce9